padd.sh

```diff
@@ -1160,7 +1160,7 @@ PrintDashboard() {
         moveXOffset; printf " %-10s%-16s %-4s%-7s %-4s%-5s${clear_line}\n" "Interfce:" "${iface_name}" "TX:" "${tx_bytes}" "RX:" "${rx_bytes}"
         moveXOffset; printf " %-10s%-16s %-8s${dnssec_heatmap}%-16s${reset_text}${clear_line}\n" "DNS:" "${dns_information}" "DNSSEC:" "${dnssec_status}"
         moveXOffset; printf " %-10s%s${clear_line}\n" "IPv6:" "${pi_ip6_addr}"
-        moveXOffset; printf " %-10s%-15s%-4s${dhcp_range_heatmap}%-36s${reset_text}${clear_line}\n" "DHCP:" "${dhcp_check_box}" "Rng" "${dhcp_range}"
+        moveXOffset; printf " %-10s%-15s${dhcp_range_heatmap}%-36s${reset_text}${clear_line}\n" "DHCP:" "${dhcp_check_box}" "${dhcp_range}"
```

Comment:
I took a deep dive into this and found the underlying issue.
It's the `check_box_disabled="[${blue_text}-${reset_text}]"` in conjunction with the `printf` command and the padding. `printf` prints padding by bytes, not characters. The check_mark or cross symbols need 3 bytes, the `-` sign only one byte. Therefore it will add extra space before the 'Rng' when the DHCP is disabled, which causes the line break due to the long padding for the IP range. 

The trick is now to decrease the padding for the symbole but add a fixed space. (And reduce the final line padding a bit.)

This is working for me, even when keeping the 'Rng'

```suggestion
        moveXOffset; printf " %-10s%-5s %-4s${dhcp_range_heatmap}%-33s${reset_text}${clear_line}\n" "DHCP:" "${dhcp_check_box}" "Rng" "${dhcp_range}"
```

Reply:
Wow, that's awesome you found the root cause of this. Nice work!